src/_includes/blog.njk

```diff
@@ -0,0 +1,156 @@
+---
+title: Blogs
+layout: layout.njk
+---
+
+<div>
+    <section class="page-header page-header-modern bg-color-quaternary p-relative">
+        <div class="container">
+            <div class="row py-5">
+                <div class="col-md-8 order-2 order-md-1 align-self-center p-static">
+                    <h1 class="text-color-dark font-weight-bold">{{ title }}</h1>
+                </div>
+                <div class="col-md-4 order-1 order-md-2 align-self-center">
+                    <ul class="breadcrumb d-flex justify-content-md-end text-3-5">
+                        <li>
+                            <a href="" class="text-color-default text-color-hover-primary text-decoration-none">HOME</a>
+                        </li>
+                        <li class="active">Blog</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <div class="container pt-4 pb-5 mb-5">
+        <div class="row">
+            {{ content | safe }}
+            <div class="blog-sidebar col-lg-4 pt-4 pt-lg-0">
+                <aside class="sidebar">
+                    {% if thumbnail %}
+                    <div class="px-3 mb-4">
+                        <h3 class="text-color-dark text-capitalize font-weight-bold text-5 m-0 mb-3">About The Blog</h3>
+                        <p class="m-0">{{description}}</p>
+                    </div>
+                    {% endif %}
+                    <div class="px-3">
+                            <div class="input-group mb-3 pb-1">
+                                <input
+                                    class="form-control box-shadow-none text-1 border-0 bg-color-grey"
+                                    placeholder="Search..."
+                                    name="search-posts"
+                                    id="search-posts"
+                                    type="search"/>
+                            </div>
+                        <ul id="searchResults" class="simple-post-list m-0"></ul>
+                        <div id="noResultsFound" style="display: none">
+                            <p style="color: #777">No results found.</p>
+                        </div>
+                        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/elasticlunr/0.9.6/elasticlunr.min.js"></script>
+                        <script type="text/javascript" src="/js/search.js"></script>
```

Comment:
Script tags in the middle of page body. This should not happen. Script tags should be loaded towards the end of body tag if they are not critical and we should `defer` their loading

Reply:
done!